frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -421,83 +475,19 @@ public void refreshModel() {
     val gitRepository = gitRepositorySelectionProvider.getSelectedGitRepository();
     if (gitRepository != null) {
       refreshModel(gitRepository,
-          NullGitMacheteRepositorySnapshot.getInstance(),
-          /* doOnUIThreadWhenReady */ () -> {});
+              NullGitMacheteRepositorySnapshot.getInstance(),
+              /* doOnUIThreadWhenReady */ () -> {});
     } else {
       LOG.warn("Selected git repository is undefined; unable to refresh model");
     }
   }
 
   @UIEffect
-  private void refreshModel(
```

Comment:
Pls move method to the previous position to focus the diff on the actual changes